src/main/java/com/simibubi/create/content/contraptions/AbstractContraptionEntity.java

```diff
@@ -78,6 +80,9 @@
 import net.minecraftforge.network.NetworkHooks;
 import net.minecraftforge.network.PacketDistributor;
 
+import static java.lang.Math.abs;
+import static java.lang.Math.signum;
```

Comment:
Static imports make code a little harder to read as you can't easily see where a method is from without scrolling up to the imports or without a IDE